### ISSUE TITLE
Error APT2144	invalid file path dotnet_bot.svg

### ISSUE THREAD

**Issue by ComptonAlvaro** [dotnet/maui#3900]
### Description

I have create a MAUI project with VS 2022 preview, and then I try to compile it.

First I get an error that was decribe in the issue [#3896](https://github.com/dotnet/maui/issues/3896).

When I rename the name from OpenSans-Regular.ttf to OpenSansRegular.ttf I get another error: Error APT2144	invalid file path dotnet_bot.svg.

Also I get another related errores:

Error APT2144	invalid file path 'C:\proyectos\GestorOrdenadores\GestorOrdenadores.Cliente.XamarinMaui\obj\Debug\net6.0-android\res\resources\appiconfg.svg'.	GestorOrdenadores.Cliente.XamarinMaui	C:\Program Files\dotnet\packs\Microsoft.Android.Sdk.Windows\31.0.101-preview.10.59\tools\Xamarin.Android.Aapt2.targets	123

Error APT2144	invalid file path 'C:\proyectos\GestorOrdenadores\GestorOrdenadores.Cliente.XamarinMaui\obj\Debug\net6.0-android\res\resources\appicon.svg'.	GestorOrdenadores.Cliente.XamarinMaui	C:\Program Files\dotnet\packs\Microsoft.Android.Sdk.Windows\31.0.101-preview.10.59\tools\Xamarin.Android.Aapt2.targets	123	

Error APT2144	invalid file path 'C:\proyectos\GestorOrdenadores\GestorOrdenadores.Cliente.XamarinMaui\obj\Debug\net6.0-android\res\resources\fonts\opensansregular.ttf'.	GestorOrdenadores.Cliente.XamarinMaui	C:\Program Files\dotnet\packs\Microsoft.Android.Sdk.Windows\31.0.101-preview.10.59\tools\Xamarin.Android.Aapt2.targets	123	

Thanks so much.

### Steps to Reproduce

1.- Create a MAUI project
2.- Compilte
3.- Rename OpenSans-Regular.ttf file to OpenSansRegular.ttf
4.- Complile.

### Version with bug

Preview 11

### Last version that worked well

Unknown/Other

### Affected platforms

Android, Windows

### Affected platform versions

Android

### Relevant log output

```shell
Operación Recompilar iniciada...
Se ha restaurado C:\proyectos\GestorOrdenadores\GestorOrdenadores.Service.Grpc.DotNet\GestorOrdenadores.Service.Grpc.DotNet.csproj (en 2,33 sec).
Se ha restaurado C:\proyectos\GestorOrdenadores\GestorOrdenadores.Cliente.Wpf\GestorOrdenadores.Cliente.Wpf.csproj (en 2,33 sec).
Se ha restaurado C:\proyectos\GestorOrdenadores\GestorOrdenadores.Service.Comun\GestorOrdenadores.Service.Comun.csproj (en 2,33 sec).
Se ha restaurado C:\proyectos\GestorOrdenadores\GestorOrdenadores.Cliente\GestorOrdenadores.Cliente.csproj (en 2,33 sec).
Se ha restaurado C:\proyectos\GestorOrdenadores\GestorOrdenadores.Cliente.XamarinMaui\GestorOrdenadores.Cliente.XamarinMaui.csproj (en 3,03 sec).
1>------ Operación Recompilar todo iniciada: proyecto: GestorOrdenadores.Cliente.XamarinMaui, configuración: Debug Any CPU ------
1>C:\Program Files\dotnet\packs\Microsoft.iOS.Sdk\15.0.101-preview.10.251\tools\msbuild\iOS\Xamarin.Shared.targets(291,41): warning MSB4130: Es posible que la condición "'$(IsMacEnabled)' == 'true' And '$(IsAppExtension)' == 'true' Or '$(_PlatformName)' == 'macOS'" se evalúe incorrectamente en una versión anterior de MSBuild. Compruebe que el orden especificado de las cláusulas AND y OR es el previsto. Para evitar esta advertencia, agregue paréntesis para definir explícitamente el orden de evaluación.
1>C:\Program Files\dotnet\packs\Microsoft.MacCatalyst.Sdk\15.0.101-preview.10.251\tools\msbuild\iOS\Xamarin.Shared.targets(291,41): warning MSB4130: Es posible que la condición "'$(IsMacEnabled)' == 'true' And '$(IsAppExtension)' == 'true' Or '$(_PlatformName)' == 'macOS'" se evalúe incorrectamente en una versión anterior de MSBuild. Compruebe que el orden especificado de las cláusulas AND y OR es el previsto. Para evitar esta advertencia, agregue paréntesis para definir explícitamente el orden de evaluación.
1>Compilación del proyecto "GestorOrdenadores.Cliente.XamarinMaui.csproj" terminada.
1>C:\Program Files\dotnet\packs\Microsoft.iOS.Windows.Sdk\15.0.101-preview.10.251\tools\msbuild\iOS\Xamarin.Messaging.Apple.targets(31,3): warning : There is no available connection to the Mac. Task 'VerifyXcodeVersion' will not be executed
1>GestorOrdenadores.Cliente.XamarinMaui -> C:\proyectos\GestorOrdenadores\GestorOrdenadores.Cliente.XamarinMaui\bin\Debug\net6.0-ios\iossimulator-x64\GestorOrdenadores.Cliente.XamarinMaui.dll
1>GestorOrdenadores.Cliente.XamarinMaui -> C:\proyectos\GestorOrdenadores\GestorOrdenadores.Cliente.XamarinMaui\bin\Debug\net6.0-maccatalyst\maccatalyst-x64\GestorOrdenadores.Cliente.XamarinMaui.dll
1>Compilación del proyecto "GestorOrdenadores.Cliente.XamarinMaui.csproj" terminada.
1>C:\Program Files\dotnet\packs\Microsoft.Android.Sdk.Windows\31.0.101-preview.10.59\tools\Xamarin.Android.Aapt2.targets(123,3): error APT2144: invalid file path 'C:\proyectos\GestorOrdenadores\GestorOrdenadores.Cliente.XamarinMaui\obj\Debug\net6.0-android\res\resources\images\dotnet_bot.svg'.
1>C:\Program Files\dotnet\packs\Microsoft.Android.Sdk.Windows\31.0.101-preview.10.59\tools\Xamarin.Android.Aapt2.targets(123,3): error APT2144: invalid file path 'C:\proyectos\GestorOrdenadores\GestorOrdenadores.Cliente.XamarinMaui\obj\Debug\net6.0-android\res\resources\appiconfg.svg'.
1>C:\Program Files\dotnet\packs\Microsoft.Android.Sdk.Windows\31.0.101-preview.10.59\tools\Xamarin.Android.Aapt2.targets(123,3): error APT2144: invalid file path 'C:\proyectos\GestorOrdenadores\GestorOrdenadores.Cliente.XamarinMaui\obj\Debug\net6.0-android\res\resources\appicon.svg'.
1>C:\Program Files\dotnet\packs\Microsoft.Android.Sdk.Windows\31.0.101-preview.10.59\tools\Xamarin.Android.Aapt2.targets(123,3): error APT2144: invalid file path 'C:\proyectos\GestorOrdenadores\GestorOrdenadores.Cliente.XamarinMaui\obj\Debug\net6.0-android\res\resources\fonts\opensansregular.ttf'.
1>Compilación del proyecto "GestorOrdenadores.Cliente.XamarinMaui.csproj" terminada -- ERROR.
1>GestorOrdenadores.Cliente.XamarinMaui -> C:\proyectos\GestorOrdenadores\GestorOrdenadores.Cliente.XamarinMaui\bin\Debug\net6.0-windows10.0.19041\win-x64\GestorOrdenadores.Cliente.XamarinMaui.dll
========== Recompilar todo: 0 correctos, 1 incorrectos, 0 omitidos ==========
```

**Comment by weltraumreise** [comment]
Could you please post your .csproj file? I have found a solution for a similar issue with my Maui project. I had a project worked before and all the new projects do not work any more without the modification of the csproj file, after I have done some updates. I have found the cause by comparing the old project and newly created project.
The MAUI dotnet templeate has been changed. Those changes lead to this issue, but I don't know which change exactly is the cause. If you could post your csproj file, I may help.

**Comment by gyretech** [comment]
MauiIcon and MauiSplashScreen both changed to AndroidResource

**Comment by ComptonAlvaro** [comment]
Thanks for the help for all of you.

This is my project file of the MAUI project.

`<Project Sdk="Microsoft.NET.Sdk">

	<PropertyGroup>
		<TargetFrameworks>net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows')) and '$(MSBuildRuntimeType)' == 'Full'">$(TargetFrameworks);net6.0-windows10.0.19041</TargetFrameworks>
		<OutputType>Exe</OutputType>
		<RootNamespace>GestorOrdenadores.Cliente.XamarinMaui</RootNamespace>
		<UseMaui>true</UseMaui>
		<SingleProject>true</SingleProject>
		<EnablePreviewMsixTooling>true</EnablePreviewMsixTooling>

		
		<ApplicationTitle>GestorOrdenadores.Cliente.XamarinMaui</ApplicationTitle>

		
		<ApplicationId>com.companyname.GestorOrdenadores.Cliente.XamarinMaui</ApplicationId>

		
		<ApplicationVersion>1</ApplicationVersion>

		
		<UseInterpreter Condition="'$(Configuration)' == 'Debug'">True</UseInterpreter>

		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-ios'">14.2</SupportedOSPlatformVersion>
		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-maccatalyst'">14.0</SupportedOSPlatformVersion>
		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-android'">21.0</SupportedOSPlatformVersion>
		<SupportedOSPlatformVersion Condition="$(TargetFramework.Contains('-windows'))">10.0.17763.0</SupportedOSPlatformVersion>
		<TargetPlatformMinVersion Condition="$(TargetFramework.Contains('-windows'))">10.0.17763.0</TargetPlatformMinVersion>
	</PropertyGroup>

	<ItemGroup>
		
		<AndroidResource Include="Resources\appicon.svg" ForegroundFile="Resources\appiconfg.svg" Color="#512BD4" />

		
		<AndroidResource Include="Resources\appiconfg.svg" Color="#512BD4" />

		
		<MauiImage Include="Resources\Images\*" />

		
		<MauiFont Include="Resources\Fonts\*" />
	</ItemGroup>

	<ItemGroup Condition="$(TargetFramework.Contains('-windows'))">
		
		<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0-preview3" />
		<PackageReference Include="Microsoft.Graphics.Win2D" Version="1.0.0.29-preview3" />
	</ItemGroup>

	<ItemGroup>
	  <MauiFont Remove="Resources\Fonts\OpenSansRegular.ttf" />
	</ItemGroup>

	<ItemGroup>
	  <MauiImage Remove="Resources\Images\dotnet_bot.svg" />
	</ItemGroup>

	<ItemGroup>
	  <AndroidResource Include="Resources\Fonts\opensansregular.ttf" />
	  <AndroidResource Include="Resources\Images\dotnet_bot.svg" />
	</ItemGroup>

	<PropertyGroup Condition="$(TargetFramework.Contains('-windows'))">
		<OutputType>WinExe</OutputType>
		<RuntimeIdentifier>win-x64</RuntimeIdentifier>
	</PropertyGroup>
	
</Project>
`



Thanks.

**Comment by weltraumreise** [comment]
Try this
`...

<ItemGroup>
	
    
	
    <MauiIcon Include="Resources\appicon.svg" ForegroundFile="Resources\appiconfg.svg" Color="#512BD4" />

	
        
	
    <MauiSplashScreen Include="Resources\appiconfg.svg" Color="#512BD4" />


	
	<MauiImage Include="Resources\Images\*" />

	
	<MauiFont Include="Resources\Fonts\*" />
</ItemGroup>

<ItemGroup Condition="$(TargetFramework.Contains('-windows'))">
	
	<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0-preview3" />
	<PackageReference Include="Microsoft.Graphics.Win2D" Version="1.0.0.29-preview3" />
</ItemGroup>




<PropertyGroup Condition="$(TargetFramework.Contains('-windows'))">
	<OutputType>WinExe</OutputType>
	<RuntimeIdentifier>win-x64</RuntimeIdentifier>
</PropertyGroup>
`

**Comment by weltraumreise** [comment]
In the first <ItemGroup> make these changes
`<ItemGroup>
	
    
	
    <MauiIcon Include="Resources\appicon.svg" ForegroundFile="Resources\appiconfg.svg" Color="#512BD4" />

	
        
	
    <MauiSplashScreen Include="Resources\appiconfg.svg" Color="#512BD4" />


	
	<MauiImage Include="Resources\Images\*" />

	
	<MauiFont Include="Resources\Fonts\*" />
</ItemGroup>`

And Remove the last three <ItemGroup>s like this
`


`

**Comment by weltraumreise** [comment]
Change `<AndroidResource Include="Resources\appicon.svg" ForegroundFile="Resources\appiconfg.svg" Color="#512BD4" />` to `<MauiIcon Include="Resources\appicon.svg" ForegroundFile="Resources\appiconfg.svg" Color="#512BD4" />`

Change `<AndroidResource Include="Resources\appiconfg.svg" Color="#512BD4" />` to `<MauiSplashScreen Include="Resources\appiconfg.svg" Color="#512BD4" />`


And remove the last three "item group"s

**Comment by ComptonAlvaro** [comment]
I delete the project and I created it again with the version 17.1.0 preview 2.0 and it compiles.

I have check the project file and the lines that you advise me to change are in this new project, so it works as expected now with the new version.


Thanks.

**Comment by jfversluis** [comment]
Assuming this was a incidental hiccup then, glad you figured it out!

**Comment by MagicAndre1981** [comment]
I get the same error with VS2022 17.1 Preview 3 when I create a **new** MAUI app. So this is a template issue @jfversluis when I need to change so many things after creating the app.

**Comment by nextfool** [comment]
> 

Doing this fixed the errors in vs2022 17.1 preview 3. Related issue https://github.com/dotnet/maui/issues/3896
